.ci/pytorch/common_utils.sh

```diff
@@ -95,6 +95,23 @@ function get_bazel() {
   chmod +x tools/bazel
 }
 
+# This function is bazel specific because of the bug
+# in the bazel that requires some special paths massaging
+# as a workaround. See
+# https://github.com/bazelbuild/bazel/issues/10167
+function install_sccache_nvcc_for_bazel() {
+  sudo mv /usr/local/cuda/bin/nvcc /usr/local/cuda/bin/nvcc-real
+
+  echo '#!/bin/sh' | sudo tee /usr/local/cuda/bin/nvcc
```

Comment:
May be cat the script here to a file, then sudo mv to `/usr/local/cuda/bin/nvcc` in one go to make this a bit easier to read?